atef/widgets/config/run_active.py

```diff
@@ -98,3 +101,80 @@ def __init__(self, *args, data: PreparedDescriptionStep, **kwargs):
     def _setup_ui(self) -> None:
         self.title_label.setText(self.bridge.origin.get().name)
         self.desc_label.setText(self.bridge.origin.get().description)
+
+
+class SetValueRunWidget(DesignerDisplay, DataWidget):
+    """
+    Widget for viewing set value step.  Displays Results alongside their
+    respective actions or checks
+    """
+    filename = 'set_value_run_widget.ui'
+
+    actions_table: QtWidgets.QTableWidget
+    checks_table: QtWidgets.QTableWidget
+
+    def __init__(self, *args, data: PreparedSetValueStep, **kwargs):
+        super().__init__(*args, data=data, **kwargs)
+        self._setup_ui()
+
```

Comment:
One more small thought on a random line: any consideration for whether it'd be more or less intuitive with the value on the left and the arrow pointing to the right? I think that mucks with the layout alignment a bit but it might be worth it

Reply:
Alternatively, keeping the ordering and replacing the arrow with text e.g. "set to"? Just so the user can read the line from left to right and make quick sense of it.

All of these random suggestions are just suggestions and I'm not particularly attached to them, but I did feel a little backward looking at this widget and trying to use it for the first time.

Reply:
I was on the fence regarding this for a bit.  I arranged it so the fields would be filled out from left to right; you have to specify a target before you can pick a value to set it to.  

I think it's pretty easy to do this either way

Reply:
Switched the direction of the arrow in [21dbf46](https://github.com/pcdshub/atef/pull/159/commits/21dbf4695b92dbbeeaee659780f8b0492b3afdac)